demos/python_demos/whiteboard_inpainting_demo/whiteboard_inpainting_demo.py

```diff
@@ -0,0 +1,185 @@
+"""
```

Comment:
Please, set execution permissions for that file and add shebang line.

Reply:
Added.